source/guides/getting-started.md

```diff
@@ -1011,8 +1012,8 @@ Open up `apps/web/templates/books/new.html.erb`:
   <div class="errors">
     <h3>There was a problem with your submission</h3>
     <ul>
-      <% params.errors.each do |error| %>
-        <li><%= error.attribute_name %> is required</li>
+      <% params.errors(full: true).each do |error| %>
```

Comment:
I was looking here for an example of how to display errors in the upcoming release, but it looks like `Hanami::Action::Params#errors` doesn't take any arguments so this hits an error. Or is there a Hanami::Controller branch that has this?